0011-command-execution-protocol/README.md

```diff
@@ -0,0 +1,170 @@
+---
+name: Command Execution Protocol
+shortname: 11/CEP
+status: raw
+editor: Alexey Shmalko <ashmalko@kaaiot.io>
+---
+
+<!-- toc -->
+
+# Introduction
+Command Execution Protocol is an endpoint-aware 1/KP extension. It is designed to allow command execution on endpoints.
```

Comment:
KP versions may be not backward compatible so that won't work. I was thinking of explicitly stating in future KP version when they are backward compatible in a way that they can substitute 1/KP.